index.html

```diff
@@ -4116,91 +4060,475 @@ <h3>Parsing content</h3>
+          <a href="https://github.com/w3c/permissions/issues/47">defined</a> as
+          "`nfc`".
+        </p>
+      </section>
+      <section> <h4><dfn>Ask for forgiveness</dfn></h4>
+        <p>
+          Denotes an unobtrusive way to notify the user about an ongoing operation
+          with a possible action to cancel or undo the operation, and to
+          handle future occurences of the same operation by the same action.
+        </p>
+      </section>
+    </section>
+
+    <section> <h3>Encrypting <a>NFC content</a></h3>
+      <p>
+        For trusting the confidentiality of the data exchanged via NFC, user agents
+        MAY use encrypted <a>NFC content</a> with key management based on
+        Public Key Infrastructure (PKI).
+        Key management is out of the scope of Web NFC.
+      </p>
+      <p class="note">
+        This is different from application level end to end encryption.
+        Key management is then in application domain.
+      </p>
+    </section>
+
+    <section> <h3>Signing NDEF records</h3>
+      <p>
+        For trusting the integrity of the data exchanged via NFC, user agents
+        MAY use an <a>NDEF signature</a> with a Public Key Infrastructure
+        for key management.
+      </p>
+      <p>
+        For tags signed with <a>NDEF signature</a> version 1.0 ([[NFC-SECURITY]]),
+        the signature is applied only to the <a>TYPE field</a>, <a>ID field</a>
+        and <a>PAYLOAD field</a>, leaving out the first byte of the NDEF header,
+        allowing surface to attacks. Version 2.0 of [[NFC-SECURITY]] included
+        tag hardware attributes in the signature and allowed for shorter
+        certificates.
+      </p>
+    </section>
+
+    <section> <h3>Content identification</h3>
+      <p>
+        Implementations or applications MAY use an identification mechanism for
+        telling apart “Web NFC content" from generic NFC content. Also, a
+        mechanism for conveying implementation specific metadata which is not
+        exposed to applications MAY be implemented.
+      </p>
+      <section> <h4>Store site URL as record identifier when writing data</h4>
+        <p>
+          When pushing an <a>NDEF message</a>, the [= host/registrable domain =],
+          <a data-cite="dom#concept-host-serializer">serialized</a>,
+          of the <a>current settings object</a> MAY be stored as the <a>record
+          identifier</a> in each top-level <a>NDEF Record</a>.
+          This does not make the payload trusted.
+        </p>
+      </section>
+      <section> <h4>Include URI records with registrable domain</h4>
+        <p>
+          When pushing an <a>NDEF message</a>, the [= host/registrable domain =],
+          <a data-cite="dom#concept-host-serializer">serialized</a>,
+          of the <a>current settings object</a> MAY be stored as a signed
+          <a>URI record</a>. This does not make other records trusted.
+        </p>
+      </section>
+    </section>
```

Comment:
It seems to me that a "real" example for content identification would be more about storing a public key in a tag and checking its authenticity with a private key stored in the cloud?

Reply:
Yes, we could add that as a separate mechanism. 

This one is the remains of an old idea whose context has changed. Currently we don't do this, but does it mean we have to remove it from the Security section (again, this is a general informal section). The normative part is in the Security policies section.

Reply:
I'd prefer leaving it out as this would give (bad) ideas to people.